src/main/resources/templates/casInstitutionLoginView.html

```diff
@@ -99,10 +99,12 @@ <h1 th:utext="#{screen.institutionlogin.heading}"></h1>
                         /*<![CDATA[*/
                             institutionLoginUrl = /*[[${pac4jInstnLoginUrls.get('cord')}]]*/ '';
                         /*]]>*/
-                    } else if (institutionLoginUrl === "fakecas") {
+                    } else if (institutionLoginUrl === "osftype0") {
                         /*<![CDATA[*/
-                            institutionLoginUrl = /*[[${pac4jInstnLoginUrls.get('fakecas')}]]*/ '';
+                            institutionLoginUrl = /*[[${pac4jInstnLoginUrls.get('osftype0')}]]*/ '';
                         /*]]>*/
+                    } else if (institutionLoginUrl === "via-orcid") {
+                        institutionLoginUrl = /*[[${osfCasLoginContext.orcidLoginUrl}]]*/ ''
```

Comment:
When the via-orcid institutions are selected, clicking the sign-in button goes to ORCiD login directly.

https://user-images.githubusercontent.com/3750414/160059831-47781a60-8c31-4959-8657-55a0dbd9dc4b.mov